src/services/userDataService.js

```diff
@@ -0,0 +1,23 @@
+export const usersList = [
```

Comment:
@lenon777 , if you are writing code within TypeScript why the file has a `.js` extension?